modules/ding_provider/connie_search/includes/connie_search.search.inc

```diff
@@ -125,7 +125,7 @@ function connie_search_search_map_common_fields() {
   return [
     TingSearchCommonFields::ACQUISITION_DATE => 'acquisitionDate',
     TingSearchCommonFields::AUTHOR => 'facet.creator',
-    TingSearchCommonFields::CATEGORY => 'facet.category',
```

Comment:
This change seems quite odd to me. Is `category` actually for audiences in OpenSearch?

Reply:
As far as I can tell from examples and from this http://opensearch.addi.dk/4.0.1/opensearch_cql.xml "category" can have two values in opensearch, 	"Voksenmaterialer" or "Børnematerialer", so audience seemed like a better name - but I just realized that we of cause also have an actual audience field - so this name is probably bad.